egs/wsj/s5/steps/nnet3/make_cnn_tdnn_configs.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/env python
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import re, os, argparse, sys, math, warnings
+import numpy as np
+
+def ComputeLifterCoeffs(Q, dim):
+    coeffs = np.zeros((dim))
+    for i in range(0, dim):
+        coeffs[i] = 1.0 + 0.5 * Q * math.sin(math.pi * i / Q);
+
+    return coeffs
+
+def ComputeIDctMatrix(K, N, cepstral_lifter=0):
+    matrix = np.zeros((K, N))
+    # normalizer for X_0
+    normalizer = math.sqrt(1.0 / N);
+    for j in range(0, N):
+        matrix[0, j] = normalizer;
+    # normalizer for other elements
+    normalizer = math.sqrt(2.0 / N);
+    for k in range(1, K):
+      for n in range(0, N):
+        matrix[k, n] = normalizer * math.cos(math.pi/N * (n + 0.5) * k);
+
+    if cepstral_lifter != 0:    
+        lifter_coeffs = ComputeLifterCoeffs(cepstral_lifter, K)
+        for k in range(0, K):
+            matrix[k, :] = matrix[k, :] / lifter_coeffs[k];
+
+    return matrix.T
+
+parser = argparse.ArgumentParser(description="Writes config files and variables "
```

Comment:
Based on @vdp 's comments I made changes to the organization of python scripts. Please see [script](https://github.com/vijayaditya/kaldi/blob/e4d6ccde9f11890de253f6c264875f0f8bdf23ef/egs/wsj/s5/steps/nnet3/lstm/make_configs.py) for example on how to write the Argument parsing code.